app/Http/Controllers/AdminDash.php

```diff
@@ -1370,15 +1371,35 @@ public function saveNewEvent(Request $request) {
             'end_time' => 'required',
             'description' => 'required'
         ]);
-
-        if ($request->file('banner') != null) {
+        
+        if ($request->file('banner') != null and $request->filled('banner_url')) {
+            return redirect()->back()->withErrors('Please ensure you submit only one of the following: a URL or a file for the banner.')->withInput();
+        } elseif ($request->file('banner') != null) {
```

Comment:
The previous if ended with a return (so if it was true, we'll never get here), so let's break this out and make it a new if statement